cranelift/codegen/src/isa/aarch64/inst.isle

```diff
@@ -1731,6 +1731,11 @@
 (decl writable_zero_reg () WritableReg)
 (extern constructor writable_zero_reg writable_zero_reg)
 
+(decl value_zero_reg () ValueRegs)
+(rule (value_zero_reg)
+      (value_regs (zero_reg) (zero_reg)))
```

Comment:
We unfortunately can't do this (right now) because the zero reg is not allocatable (and if it were, we are also working on removing "pinned vregs", where a vreg refers to a specific physical reg always); @elliottt is working on using fixed-reg operands to allow this in a better way. For now, can we use `(imm $I64 0)` for zeroes (and then we can apply the "use zero reg for a zero" optimization in one place later)?